src/components/manage_dac_table/ManageDacTable.js

```diff
@@ -0,0 +1,249 @@
+import {Fragment, useState, useEffect, useCallback} from 'react';
+import { h } from 'react-hyperscript-helpers';
+import ReactTooltip from 'react-tooltip';
+import {DAC} from '../../libs/ajax';
+import {filter, isNil} from 'lodash/fp';
+import {Styles} from '../../libs/theme';
+import { recalculateVisibleTable, goToPage as updatePage } from '../../libs/utils';
+import cellData from './ManageDacTableCellData';
+import SimpleTable from '../../components/SimpleTable';
+import PaginationBar from '../../components/PaginationBar';
+
+
+export const styles = {
+  baseStyle: {
+    fontFamily: 'Montserrat',
+    fontSize: '1.4rem',
+    fontWeight: 400,
+    display: 'flex',
+    padding: '1rem 2%',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    whiteSpace: 'pre-wrap',
+    backgroundColor: 'white',
+    border: '1px solid #DEDEDE',
+    borderRadius: '4px',
+    margin: '0.5% 0',
+    overflow: 'visible'
+  },
+  columnStyle: Object.assign({}, Styles.TABLE.HEADER_ROW, {
+    justifyContent: 'space-between',
+    color: '#7B7B7B',
+    fontFamily: 'Montserrat',
+    fontSize: '1.2rem',
+    fontWeight: 'bold',
+    letterSpacing: '0.2px',
+    textTransform: 'uppercase',
+    backgroundColor: 'B8CDD3',
+    border: 'none',
+  }),
+  cellWidth: {
+    name: '25%',
+    description: '62%',
+    datasets: '10%',
+    actions: '3%',
```

Comment:
Optional nit: you could give a little more space for actions so that the page can handle smaller screen widths a bit better. For me the action icons are at the edge at ~1100px and then start to go outside the table by 1000px